gsoc/ideas-2018.html

```diff
@@ -59,7 +59,7 @@ <h1>Google Summer of Code 2018 Idea List</h1>
         </p>
         <p class="center">
           <a class="ui big inverted red button"
-             href="http://openwisp.io/docs/developer/google-summer-of-code.html">
```

Comment:
I think this is not good because if we move the site to HTTPS all links will not work because not every link is available via HTTPs. This applies for all changes similar to this one. Can you find a way to avoid doing this? If necessary, use a different linter as mentioned before.

Reply:
That is definitely not the correct way of doing this. I was just trying to avoid warnings to see if it works but now i will try a different linter as this one doesn't allow a customisable configuration file.